Basic-Car-Maintenance/Shared/Odometer/Views/AddOdometerReadingView.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  AddOdometerReadingView.swift
+//  Basic-Car-Maintenance
+//
+//  Created by Nate Schaffner on 10/15/23.
+//
+
+import SwiftUI
+
+struct AddOdometerReadingView: View {
+    
+    let vehicles: [Vehicle]
+    let addTapped: (OdometerReading) -> Void
+
+    @State private var date = Date()
+    @State private var selectedVehicle: Vehicle?
```

Comment:
selectedVehicle isn't needed as a state property, you only need it declared in the button action